src/current/v23.1/sso-sql.md

```diff
@@ -1,158 +1,265 @@
 
-	- **CockroachDB {{ site.data.products.cloud }}**: You must have access to a [CockroachDB {{ site.data.products.dedicated }}cluster](https://www.cockroachlabs.com/docs/cockroachcloud/create-your-cluster).
+{{site.data.alerts.callout_success}}
+This example uses [`cockroach sql`]({% link {{ page.version.version }}/cockroach-sql.md %}), but you can use any SQL client that supports sufficiently long passwords.
+{{site.data.alerts.end}}
 
-	- SQL users/credentials:
+1. Obtain a token.
 
-		- Your SQL user must have the ability to update cluster settings. This permission is provided by either the [`admin` role]({% link {{ page.version.version }}/security-reference/authorization.md %}#admin-role) or the [`MODIFYCLUSTERSETTING` role option]({% link {{ page.version.version }}/security-reference/authorization.md %}#supported-privileges). This is required to designate an IdP as an external token issuer.
+    Go to your cluster's DB Console and click the **Generate JWT auth token for cluster SSO** button
 
-		- A SQL user that corresponds with your external identity must be pre-provisioned on the cluster. To provision such users, you must have access to the [`admin` role]({% link {{ page.version.version }}/security-reference/authorization.md %}#admin-role).
+1. Use the token in place of a password in your database connection string.
 
-## Learn more
+    {% include_cached copy-clipboard.html %}
+    ~~~shell
+    cockroach sql --url "postgresql://{SQL_USERNAME}:{JWT_TOKEN}@{CLUSTER_HOST}:26257?options=--crdb:jwt_auth_enabled=true" --certs-dir={CLUSTER_CERT_DIR}
+    ~~~
 
-This [Cockroach Labs blog post](https://www.cockroachlabs.com/blog/sso-to-clusters-with-jwt/) covers and provides further resources for a variety of auth token-issuing use cases, including using Okta and Google Cloud Platform to issue tokens.
+    ~~~txt
+    Welcome to the cockroach SQL interface...
+    ~~~
 
-## Provision a service account
+## Configure your cluster for SSO
 
-For access to a CockroachDB cluster, an identity in an external IdP must correspond to a SQL user on the target cluster, so creating a service for your cluster entails two steps:
+**Prerequisites**:
 
-- Create a service account/IAM username with your external IdP (for example, GCP).
-- [Create a SQL user]({% link {{ page.version.version }}/create-user.md %}) in your cluster.
+You must have the ability to update your cluster settings, which can be achieved in several ways. Refer to [`SET CLUSTER SETTING`: Required permissions](set-cluster-setting.html#required-privileges)
 
-The **correspondence** between IdP identity and SQL username is crucial. This determined by [your cluster's identity mapping](#configure-your-clusters-identity-mapping).
+### Cluster Settings
```

Comment:
Before this section, what about adding the same paragraph about using the "Update cluster settings" section?

Reply:
Sorry, I don't know what paragraph you mean.